vue/sbc-common-components/src/store/modules/notification.ts

```diff
@@ -0,0 +1,77 @@
+import { Action, Module, Mutation, VuexModule } from 'vuex-module-decorators'
+import { Notifications } from '../../models/notification'
+import NotificationService from '../../services/notification.services'
+import ConfigHelper from '../../util/config-helper'
+import { SessionStorageKeys } from '../../util/constants'
+
+@Module({
+  name: 'notification',
+  namespaced: true
+})
+export default class NotificationModule extends VuexModule {
+  notifications: Notifications = []
+  notificationCount = 0
+  notificationUnreadPriorityCount = 0
+  notificationUnreadCount = 0
```

Comment:
It would be better to use getters instead of accessing a module's state directly (as you do in SbcHeader.vue).

But, this project uses state a lot, so 🤷‍♂️

Reply:
You can probably ignore this conversation. I see that SbcHeader.vue uses both `mapState()` and `mapGetters()` to access variables in the different store modules, so I guess it doesn't matter one way or the other which you use. (The code pattern has already been set in this case.)